hopp/utilities/utilities.py

```diff
@@ -1,6 +1,6 @@
 import os
 import yaml
-
+import dill
```

Comment:
This isn't a current dependency of HOPP, though `multiprocessing-on-dill` is, could you add `dill` since it's clear it should have already been there?